### PR TITLE
feat(gif): add loop count control

### PR DESCRIPTION
**Pull request by lorenzo-arena**
Add the following utility functions for the GIF class:
- check the load state for the internal GIF
- set the loop count for the GIF
- get the loop count for the GIF

### CODE REVIEW

File: src/libs/gif/lv_gif.c [hunk 1]
```diff
@@ -137,6 +137,36 @@ void lv_gif_resume(lv_obj_t * obj)
...
 }
 
+bool lv_gif_loaded(lv_obj_t * obj)
+{
+    lv_gif_t * gifobj = (lv_gif_t *) obj;
+
+    return (gifobj->gif != NULL);
+}
+
+int lv_gif_get_loop_count(lv_obj_t * obj)
```
Comment (FASTSHIFT) [inline src/libs/gif/lv_gif.c]: Use `int32_t` type instead of int, consistent with loop_count data type.

**Review by FASTSHIFT — CHANGES_REQUESTED** [review]

File: src/libs/gif/lv_gif.c [hunk 2]
```diff
@@ -137,6 +137,36 @@ void lv_gif_resume(lv_obj_t * obj)
...
+    lv_gif_t * gifobj = (lv_gif_t *) obj;
+
+    if(gifobj->gif == NULL) {
+        return -1;
+    }
+
+    return gifobj->gif->loop_count;
+}
+
+void lv_gif_set_loop_count(lv_obj_t * obj, int count)
```
Comment (FASTSHIFT) [inline src/libs/gif/lv_gif.c]: int32_t count

**Review by FASTSHIFT — CHANGES_REQUESTED** [review]

File: src/libs/gif/lv_gif.c (hunk 1 above)
Comment (lorenzo-arena) [reply src/libs/gif/lv_gif.c]: Solved with a fixup, thanks

File: src/libs/gif/lv_gif.c (hunk 2 above)
Comment (lorenzo-arena) [reply src/libs/gif/lv_gif.c]: Solved with a fixup, thanks

File: src/libs/gif/lv_gif.c [hunk 3]
```diff
@@ -137,6 +137,36 @@ void lv_gif_resume(lv_obj_t * obj)
     lv_timer_resume(gifobj->timer);
 }
 
+bool lv_gif_loaded(lv_obj_t * obj)
```
Comment (kisvegabor) [inline src/libs/gif/lv_gif.c]:
```suggestion
bool lv_gif_is_loaded(lv_obj_t * obj)
```

**Review by kisvegabor — COMMENTED** [review]
Thank you, just a minor thing.

File: src/libs/gif/lv_gif.c (hunk 3 above)
Comment (lorenzo-arena) [reply src/libs/gif/lv_gif.c]: No problem, fixed that

**Comment by lorenzo-arena** [comment]
The "C/C++ CI / Build Windows MSVC" test seems to be failing but I don't think it's due to my changes

**Comment by kisvegabor** [comment]
Thanks! I'm re-running the CI. I hope it helps :smile:

**Review by FASTSHIFT — APPROVED** [review]

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**